### ISSUE TITLE
[Bug] VITS LJSpeech recipe no improvement in 70k steps (batch size 16)

### ISSUE THREAD

**Issue by fijipants** [coqui-ai/TTS#938]
**Describe the bug**

I've used the VITS LJSpeech recipe for 70k steps (batch size 16) and have seen no drops in loss, the alignment is always perfect, and the audio always sounds the same.

Loss:

![image](https://user-images.githubusercontent.com/88913682/142049695-6c7b92b1-a3a3-47ea-ad0e-5936f4acebe0.png)

Alignment at 12k:

![image](https://user-images.githubusercontent.com/88913682/142049705-f4ebd337-e3ab-4d9b-b441-c56a1690ffa8.png)

Alignment at 70k:

![image](https://user-images.githubusercontent.com/88913682/142049712-38c5b722-d2fc-4497-95c4-cb5ae18b4cfb.png)

Audio at 12k:

https://user-images.githubusercontent.com/88913682/142049725-c539f76b-2e10-41ef-a6e8-c29d0798738a.mp4

Audio at 70k:

https://user-images.githubusercontent.com/88913682/142049732-801815eb-886f-4b30-bb85-810fd471c6b9.mp4

**To Reproduce**

Apply the fix described in Additional context below.

Copy `recipes/ljspeech/vits_tts/train_vits.py` to `runs2/train_vits.py` and make the following changes:
- Fix ljspeech path
- batch_size=16
- eval_batch_size=8

Run it with `CUDA_VISIBLE_DEVICES=1 python runs2/train_vits.py`.

**Expected behavior**

I expect that:
- Loss drops over time.
- Alignment starts out blurry and develops a line over time.
- The audio changes over time.

**Environment (please complete the following information):**

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **PyTorch or TensorFlow version (use command below)**: PyTorch 1.10
- **Python version**: 3.7
- **CUDA/cuDNN version**: 11.3 / 8200
- **GPU model and memory**: RTX 3080 10GB
- **Exact command to reproduce**: `CUDA_VISIBLE_DEVICES=1 python runs2/train_vits.py`

**Additional context**

When I originally ran it I ran into this bug: https://github.com/NVIDIA/apex/issues/694

And I applied this fix: https://github.com/NVIDIA/apex/issues/694#issuecomment-918833904

**Comment by khusainovaidar** [comment]
Had more or less the same issue, 'solved' by turning mixed_precision to False in config.

**Comment by skol101** [comment]
Looks like apex is supported. Too bad it doesn't run on >python3.9. 
With mixed_precision: False suddenly 90% of 3090 memory is used when running batch 32, instead of 65-70%.

**Comment by skol101** [comment]
@fijipants thanks for pointing to that fix. I decided to downgrade python to 3.8 just to test out how training with apex would work. Did you manage to install C++ ext to a conda env, if you using it?

**Comment by skol101** [comment]
With mixed_precision: True after training for 150k steps , the best model was on step 87k. So something's amiss with this setting for VITS model.

**Comment by fijipants** [comment]
@khusainovaidar @skol101 Thanks for the updates.

@skol101 I'm glad to hear you had some success. To answer your questions, I'm on Python 3.7 and I haven't tried the C++ ext.

I'll try out a run with `mixed_precision: False` and `batch_size: 32` on a 3090.
